cli/operations.py

```diff
@@ -122,10 +122,20 @@ def download(config, num_images, strategy=None):
     response.raise_for_status()
 
     json_resp = response.json()
-    count = len(json_resp['imageUrls'])
+    count = len(json_resp['images'].keys())
 
     print("Received " + str(count) + " files.")
 
+    image_id_to_image_url = {}
```

Comment:
@samiyaakhtar I personally understand what is going on here but I wonder if we should create a method for this or add a comment so that one can follow more quickly. Up to you

Reply:
Yeah I can clean this up